src/Orleans.Runtime/Streams/PersistentStream/PersistentStreamPullingManager.cs

```diff
@@ -30,7 +30,7 @@ internal class PersistentStreamPullingManager : SystemTarget, IPersistentStreamP
         private readonly IQueueAdapterCache queueAdapterCache;
         private IStreamQueueBalancer queueBalancer;
         private readonly IQueueAdapterFactory adapterFactory;
-        private PersistentStreamProviderState managerState;
+        private RunState managerState;
```

Comment:
RunState? should just state be sufficient here ? What does run imply here ?

Reply:
it replaces PersistentStreamProviderState and tracks if the provider is running or not.